cloudmarker/stores/mongodbstore.py

```diff
@@ -104,3 +126,7 @@ def done(self):
         # Flush all the residual records buffers to mongodb collections
         for record_type, records in self._buffer.items():
             self._flush(record_type, records)
+
+        # Close the mongo db connection when done is called as this will be the
```

Comment:
Replace "mongo db" with "MongoDB".

Reply:
Fixed: [updated to MongoDB](https://github.com/cloudmarker/cloudmarker/pull/32/files#diff-508ea730766e9a753ad4ce9b65ec1da4R105)